docs/en/shared/integrations-quick-guide.asciidoc

```diff
@@ -30,8 +29,8 @@ for the configuration you just specified. If an {agent} is already assigned to
 the policy, you're done. Otherwise, you need to deploy an {agent}.
 . To deploy an {agent}:
 .. In the popup, click **Add {agent} to your hosts** to open the **Add agent**
-flyout. If you accidentally close the popup, go to **{fleet} -> Agents**, then
-click **Add agent** to access the flyout.
+flyout. If you accidentally close the popup or the flyout doesn't open, go to
```

Comment:
Added because clicking this button to open the flyout does not work in the latest snapshot for 8.8